src/patternfly/components/Badge/badge.scss

```diff
@@ -5,7 +5,8 @@
   --pf-c-badge--BorderRadius: var(--pf-global--BorderRadius--lg);
   --pf-c-badge--FontSize: var(--pf-global--FontSize--xs);
   --pf-c-badge--FontWeight: var(--pf-global--FontWeight--bold);
-  --pf-c-badge--PaddingX: var(--pf-global--spacer--sm);
+  --pf-c-badge--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-badge--PaddingRight: var(--pf-global--spacer--sm);
```

Comment:
Maybe put the PaddingRight variable above PaddingLeft to align with the order they're used below.

Reply:
updated!